aeon/dj_pipeline/analysis/block_analysis.py

```diff
@@ -309,18 +309,23 @@ def make(self, key):
         # Get all unique subjects that visited the environment over the entire exp;
         # For each subject, see 'type' of visit most recent to start of block
         # If "Exit", this animal was not in the block.
+        subject_names = (acquisition.Experiment.Subject & key).fetch("subject", order_by="subject").tolist()
```

Comment:
```suggestion
        subject_names = (acquisition.Experiment.Subject & key).fetch("subject", order_by="subject")
```
If you decide to adopt the below suggestion, we don't need to convert this into a list